src/collections/_documentation/workflow/alerts-notifications/alerts.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Alerts
+sidebar_order: 1
+---
+
+Sentry users with admin permissions or higher can create one of two types of alerts: [Metric Alerts](#metric-alerts) or [Issue Alerts](#issue-alerts). You can use [Metric Alerts](#metric-alerts) for high-level monitoring of patterns, or fine-grained monitoring of individual events. An [Issue Alert](#issue-alerts) fires whenever any issue in the project matches specified criteria, such as a resolved issue re-appearing or an issue affecting many users. To confirm or set admin permissions, see **Project Settings > Alerts > New Alert**. 
```

Comment:
> To confirm or set admin permissions, see **Project Settings > Alerts > New Alert**

Is this the right place to confirm/set admin permissions?

Reply:
Yep. 😸 Just went through the flow.